### DISCUSSION TITLE
List of available voice commands via Siri / Apple Watch

### DISCUSSION THREAD

**Discussion by IOHomeSol** [Kemmey/Kemmey-TeslaWatch-Public#374] [Q&A]
Great App! I researched and tried many different Tesla apps compatible with the Apple Watch, so far "Watch for Tesla" is the best one (features/price/developer). 
Can someone compile all the available/current voice commands for Watch for Tesla? It would be awesome if it was also available as a section on the official webpage (https://www.watchfortesla.com/).

Voice commands that I have tried and WORKED so far are:
1. Unlock Car
2. Lock Car
3. How many miles do I have left on my car? 

Voice commands that I have tried and NOT WORKED so far are:
1. Open windows
2. Vent windows
3. Open Frunk
4. Turn Sentry Mode Off

Thank you for considering my request.

**Comment by Kemmey** [comment]
Hey,

The commands you've found are the built-in apple car related commands, and will work without configuration. These are also implemented by the Tesla app in recent releases, so they don't even require this app to work. 

However, there's a ton of user-customizable commands where you can define your own activation phrases. 

On your phone, open the app, go to settings then tap Siri Shortcuts for a list of all available commands. Tap on any command to customize the activation phrase and add it to Siri. 

Once configured on your phone, these commands will also work on your watch - independently from your phone. 

Cheers,

Kim

**Reply by diasigvard** [reply]
So does "open trunk" work?

**Reply by calvarez** [reply]
Yes.  I can't recall if it was built in, or I added it with the shortcuts list of available commands.
